database/001_initial_schema.sql

```diff
@@ -42,5 +42,10 @@ CREATE TABLE IF NOT EXISTS public.position_log (
 
 CREATE TABLE IF NOT EXISTS config (
   conf_id SERIAL PRIMARY KEY NOT NULL,
-  threshold TEXT NOT NULL
+  p1 INTEGER NOT NULL,
+  p2 INTEGER NOT NULL,
+  p3 INTEGER NOT NULL,
+  p4 INTEGER NOT NULL,
+  p5 INTEGER NOT NULL,
+  p6 INTEGER NOT NULL
```

Comment:
Is this supposed only to serve as placeholder columns for now?

Reply:
Indeed